src/components/LaunchFilter/services/fetchLaunchConfig.ts

```diff
@@ -0,0 +1,33 @@
+import { Endpoints } from '../../../shared/api/constants/endpoints';
+import { LocationIds, LaunchFilters } from '../constants/filerConstants';
+import type { FetchLaunchConfig } from '../types/FetchLaunchConfig';
+
+export const fetchLaunchConfig = (filterValue: string): FetchLaunchConfig => {
+  switch (filterValue) {
+    case LaunchFilters.PREVIOUS:
+      return {
+        url: Endpoints.PREVIOUS_LAUNCH_ENDPOINT,
+      };
```

Comment:
why don't have `params` here?

Reply:
no need additional params, default ordering is ok